docs/02-formalia/titlepage.rst

```diff
@@ -169,6 +169,14 @@ Support available at:
 * Email: support@eprosima.com
 * Phone: +34 91 804 34 48
 
+Contributing to the documentation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+*Fast DDS-Docs* is an open source project, and as such all contributions, both in the form of feedback and content
+generation, are most welcomed.
+To make such contributions, please refer to the Contribution Guidelines hosted in our
+`Fast DDS-Docs <https://github.com/eProsima/Fast-RTPS-docs>`_ GitHub repository.
```

Comment:
I'd put the link in the *GitHub repository*, either putting everything in or just the *GitHub repository*.
```suggestion
`Fast DDS-Docs GitHub repository<https://github.com/eProsima/Fast-RTPS-docs>`_.
```

```suggestion
Fast DDS-Docs `GitHub repository<https://github.com/eProsima/Fast-RTPS-docs>`_.
```

Reply:
I found a middle ground I think